changes/2020-06-04_how-to-fail-with-keyrings/change.md

```diff
@@ -113,111 +113,147 @@ in different environments and implementations.
 ## Guide-level Explanation
 
 On encryption,
-a keyring creates an artifact:
-one or more encrypted data keys.
-This artifact has certain requirements that
-MUST be met on decryption
-in order to obtain the plaintext data key.
-This set of requirements,
-manifested as the artifact,
-is what we call the "keyring decryption contract."
+a keyring creates one or more "decryption artifacts",
+sometimes called "encrypted data keys",
+that are included in the resulting encrypted message.
+On decryption,
+a keyring consumes a decryption artifact
+and fulfills a set of requirements
+by using the decryption artifact
```

Comment:
```suggestion
a keyring consumes decryption artifacts
and fulfills a set of requirements
by using these decryption artifacts
```

at a minimum the keyring gets multiple...

As I keep reading the plural singular is everywhere.
Maybe leaving it all singular is better.
But I don't want to require a 1:1 relationship.